sdk/src/main/java/com/relaypro/sdk/Relay.java

```diff
@@ -429,6 +429,47 @@ public String translate(String text, LanguageType from, LanguageType to) {
         return null;
     }
 
+    @SuppressWarnings("unused")
+    public String placeCall(String target, String call_id) {
```

Comment:
I think the call_id argument is supposed to be the receiving device. Here is how I did it in python:
```
dest = relay.workflow.device_name("ram")

@hello_workflow.on_start
async def start_handler(relay, trigger):
    target = relay.make_target_uris(trigger)
    await relay.start_interaction(target, interaction_name)

@hello_workflow.on_interaction_lifecycle
async def lifecycle_handler(relay, itype, interaction_uri, reason):
    if itype == 'started':
        call_id = await relay.place_call(interaction_uri, dest)
    if itype == 'ended':
        await relay.hangup_call(interaction_uri, call_id)
        await relay.terminate()
```

Reply:
Ah okay - fixed in latest commit